### ISSUE TITLE
[1.8.9] Crashing after loading schematic

### ISSUE THREAD

**Issue by Jeremy-Laier** [Lunatrius/Schematica#489]
```
---- Minecraft Crash Report ----

WARNING: coremods are present:
Contact their authors BEFORE contacting forge

// Hi. I'm Minecraft, and I'm a crashaholic.

Time: 4/15/20 12:00 PM
Description: Unexpected error

java.nio.BufferOverflowException
	at java.nio.Buffer.nextPutIndex(Buffer.java:521)
	at java.nio.DirectIntBufferU.put(DirectIntBufferU.java:297)
	at net.optifine.render.VboRegion.drawArrays(VboRegion.java:310)
	at net.minecraft.client.renderer.vertex.VertexBuffer.func_177358_a(VertexBuffer.java:54)
	at com.github.lunatrius.schematica.client.renderer.chunk.container.SchematicChunkRenderContainerVbo.func_178001_a(SchematicChunkRenderContainerVbo.java:28)
	at com.github.lunatrius.schematica.client.renderer.RenderSchematic.renderBlockLayer(RenderSchematic.java:748)
	at com.github.lunatrius.schematica.client.renderer.RenderSchematic.func_174977_a(RenderSchematic.java:739)
	at com.github.lunatrius.schematica.client.renderer.RenderSchematic.renderWorld(RenderSchematic.java:353)
	at com.github.lunatrius.schematica.client.renderer.RenderSchematic.renderSchematic(RenderSchematic.java:247)
	at com.github.lunatrius.schematica.client.renderer.RenderSchematic.onRenderWorldLast(RenderSchematic.java:216)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_16_RenderSchematic_onRenderWorldLast_RenderWorldLastEvent.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:49)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:140)
	at net.minecraftforge.client.ForgeHooksClient.dispatchRenderLast(ForgeHooksClient.java:123)
	at sun.reflect.GeneratedMethodAccessor40.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.optifine.reflect.Reflector.callVoid(Reflector.java:532)
	at net.minecraft.client.renderer.EntityRenderer.func_175068_a(EntityRenderer.java:1921)
	at net.minecraft.client.renderer.EntityRenderer.func_78471_a(EntityRenderer.java:1580)
	at net.minecraft.client.renderer.EntityRenderer.func_181560_a(EntityRenderer.java:1372)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1051)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:349)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at java.nio.Buffer.nextPutIndex(Buffer.java:521)
	at java.nio.DirectIntBufferU.put(DirectIntBufferU.java:297)
	at net.optifine.render.VboRegion.drawArrays(VboRegion.java:310)
	at net.minecraft.client.renderer.vertex.VertexBuffer.func_177358_a(VertexBuffer.java:54)
	at com.github.lunatrius.schematica.client.renderer.chunk.container.SchematicChunkRenderContainerVbo.func_178001_a(SchematicChunkRenderContainerVbo.java:28)
	at com.github.lunatrius.schematica.client.renderer.RenderSchematic.renderBlockLayer(RenderSchematic.java:748)
	at com.github.lunatrius.schematica.client.renderer.RenderSchematic.func_174977_a(RenderSchematic.java:739)
	at com.github.lunatrius.schematica.client.renderer.RenderSchematic.renderWorld(RenderSchematic.java:353)
	at com.github.lunatrius.schematica.client.renderer.RenderSchematic.renderSchematic(RenderSchematic.java:247)
	at com.github.lunatrius.schematica.client.renderer.RenderSchematic.onRenderWorldLast(RenderSchematic.java:216)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_16_RenderSchematic_onRenderWorldLast_RenderWorldLastEvent.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:49)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:140)
	at net.minecraftforge.client.ForgeHooksClient.dispatchRenderLast(ForgeHooksClient.java:123)
	at sun.reflect.GeneratedMethodAccessor40.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.optifine.reflect.Reflector.callVoid(Reflector.java:532)
	at net.minecraft.client.renderer.EntityRenderer.func_175068_a(EntityRenderer.java:1921)
	at net.minecraft.client.renderer.EntityRenderer.func_78471_a(EntityRenderer.java:1580)

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [EntityPlayerSP['XArticSpartanX'/1512, l='MpServer', x=-2199.06, y=174.06, z=25560.68]]
	Chunk stats: MultiplayerChunkCache: 288, 288
	Level seed: 0
	Level generator: ID 00 - default, ver 1. Features enabled: false
	Level generator options: 
	Level spawn location: 448.00,75.00,447.00 - World: (448,75,447), Chunk: (at 0,4,15 in 28,27; contains blocks 448,0,432 to 463,255,447), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 164353254 game time, 48000 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
	Forced entities: 3 total; [EntityWither['§fOpen §e§l/token shop§f to buy §ckits§f, §akeys§f and§b collectables§f!'/1, l='MpServer', x=-2164.19, y=165.97, z=25574.06], EntityPlayerSP['XArticSpartanX'/1512, l='MpServer', x=-2199.06, y=174.06, z=25560.68], EntityItem['item.tile.sand.default'/9834395, l='MpServer', x=-2226.13, y=124.00, z=25565.88]]
	Retry entities: 0 total; []
	Server brand: BungeeCord (Bungeecord-Mineheroes-master_B51) <- TacoSpigot
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:402)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2536)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:378)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.8.9
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1206970576 bytes (1151 MB) / 1577058304 bytes (1504 MB) up to 2147483648 bytes (2048 MB)
	JVM Flags: 8 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	IntCache: cache: 0, tcache: 0, allocated: 13, tallocated: 95
	FML: MCP 9.19 Powered by Forge 11.15.1.2318 Optifine OptiFine_1.8.9_HD_U_L5 5 mods loaded, 5 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJA	mcp{9.19} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJA	FML{8.0.99.99} [Forge Mod Loader] (forge-1.8.9-11.15.1.2318-1.8.9.jar) 
	UCHIJA	Forge{11.15.1.2318} [Minecraft Forge] (forge-1.8.9-11.15.1.2318-1.8.9.jar) 
	UCHIJA	LunatriusCore{1.1.2.32} [LunatriusCore] (LunatriusCore-1.8.9-1.1.2.32-universal.jar) 
	UCHIJA	Schematica{1.7.7.143} [Schematica] (Schematica-1.8.9-1.7.7.143-universal (1).jar) 
	Loaded coremods (and transformers): 
	GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.6.0 NVIDIA 441.66' Renderer: 'GeForce GTX 960/PCIe/SSE2'
	Launched Version: 1.8.9-forge1.8.9-11.15.1.2318-1.8.9
	LWJGL: 2.9.4
	OpenGL: GeForce GTX 960/PCIe/SSE2 GL version 4.6.0 NVIDIA 441.66, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: 
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	CPU: 4x Intel(R) Core(TM) i5-2500K CPU @ 3.30GHz
	OptiFine Version: OptiFine_1.8.9_HD_U_L5
	OptiFine Build: 20191204-143016
	Render Distance Chunks: 10
	Mipmaps: 4
	Anisotropic Filtering: 1
	Antialiasing: 0
	Multitexture: false
	Shaders: null
	OpenGlVersion: 4.6.0 NVIDIA 441.66
	OpenGlRenderer: GeForce GTX 960/PCIe/SSE2
	OpenGlVendor: NVIDIA Corporation
	CpuCount: 4
```
[2.3255FusionCompact_Raiderpint_WATERED.txt](https://github.com/Lunatrius/Schematica/files/4482670/2.3255FusionCompact_Raiderpint_WATERED.txt)
This is the schematic file. Change to .schematic to load it.
